diff --git a/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js b/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
index 252ee33..607ba7b 100644
--- a/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
+++ b/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
@@ -36,7 +36,10 @@ module.exports = (packageRoot = require('./package-path'), inLernaProject) => {
       if (isLernaProject) {
         scripts[scriptName] = `lerna run ${scriptName}`;
       } else {
-        let scriptArgs = scriptFile.split('-');
+        let scriptArgs = [];
+        if (scriptFile) {
+          scriptArgs = scriptFile.replace(/\-/, '#').split('#');
+        }
         if (scriptArgs.length > 1) {
           const scriptFileName = scriptArgs[0];
           scriptArgs = scriptArgs.splice(1).join(' ');
diff --git a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
index 36c9e8c..48e6417 100644
--- a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
+++ b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
@@ -11,14 +11,24 @@ const SfdxDevConfig = require('./sfdx-dev-config');
 
 module.exports = (projectPath, inLernaProject) => {
   const pjson = new PackageJson(projectPath);
+  const pjson_devconfig = new PackageJson(process.cwd() + '/node_modules/@salesforce/dev-config');
 
   log(`Checking dependencies for ${pjson.name}`);
 
   const config = new SfdxDevConfig(projectPath);
   const dependencies = pjson.get('devDependencies');
+  const dependencies_devconfig = pjson_devconfig.get('dependencies');
 
   const added = [];
   const removed = [];
+  const synced = [];
+
+  const sync = name => {
+    if (dependencies[name] !== dependencies_devconfig[name]) {
+      synced.push(name + ' ' + dependencies[name] + ' => ' + dependencies_devconfig[name]);
+      dependencies[name] = dependencies_devconfig[name];
+    }
+  };
 
   const remove = name => {
     if (dependencies[name]) {
@@ -40,12 +50,15 @@ module.exports = (projectPath, inLernaProject) => {
 
   // Included by dev-scripts
   remove('nyc');
-  remove('typescript');
+  // remove('typescript');
+  sync('typescript')
   remove('ts-node');
   remove('@commitlint/cli');
   remove('@commitlint/config-conventional');
   remove('source-map-support');
-  remove('tslint');
+  // remove('tslint');
+  sync('tslint');
+  sync('tslint-microsoft-contrib');
   remove('xunit-file');
   remove('sinon');
   remove('mocha');
@@ -65,6 +78,11 @@ module.exports = (projectPath, inLernaProject) => {
     added.forEach(dep => log(dep, 2));
   }
 
+  if (synced.length > 0) {
+    log(`synced`);
+    synced.forEach(dep => log(dep, 2));
+  }
+
   pjson.write();
-  return added.length > 0;
+  return added.length > 0 || synced.length > 0;
 };
