patch-package
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
@@ -10,14 +10,13 @@ const shell = require('../utils/shelljs');
 
 const exists = require('../utils/exists');
 const packageRoot = require('../utils/package-path');
-const tsc = require.resolve('typescript/bin/tsc');
 
-shell.exec(`${tsc} -p . --pretty`, {
+shell.exec('tsc -p . --pretty', {
   cwd: packageRoot
 });
 
 if (exists('./test')) {
-  shell.exec(`${tsc} -p ./test --pretty`, {
+  shell.exec('tsc -p ./test --pretty', {
     cwd: packageRoot
   });
 }
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
@@ -11,12 +11,10 @@ const shell = require('../utils/shelljs');
 const packageRoot = require('../utils/package-path');
 const SfdxDevConfig = require('../utils/sfdx-dev-config');
 
-const nyc = require.resolve('nyc/bin/nyc');
-
 const config = new SfdxDevConfig(packageRoot);
 const testConfig = config.get('test') || {};
 
-let command = `${nyc} mocha`;
+let command = 'nyc mocha';
 
 if (!testConfig.mochaOpts) {
   command +=
