diff --git a/node_modules/@oclif/dev-cli/lib/commands/readme.js b/node_modules/@oclif/dev-cli/lib/commands/readme.js
index e757385..0e851d2 100644
--- a/node_modules/@oclif/dev-cli/lib/commands/readme.js
+++ b/node_modules/@oclif/dev-cli/lib/commands/readme.js
@@ -51,25 +51,28 @@ class Readme extends command_1.Command {
         return readme.replace(`<!-- ${tag} -->`, `<!-- ${tag} -->\n${body}\n<!-- ${tag}stop -->`);
     }
     toc(__, readme) {
-        return readme.split('\n').filter(l => l.startsWith('# '))
-            .map(l => l.trim().slice(2))
+        return readme.split('\n').filter(l => l.startsWith('## '))
+            .map(l => l.trim().slice(3))
             .map(l => `* [${l}](#${slugify.slug(l)})`)
             .join('\n');
     }
     usage(config) {
-        return [
-            `\`\`\`sh-session
-$ npm install -g ${config.name}
-$ ${config.bin} COMMAND
-running command...
-$ ${config.bin} (-v|--version|version)
-${config.name}/${process.env.OCLIF_NEXT_VERSION || config.version} ${process.platform}-${process.arch} node-v${process.versions.node}
-$ ${config.bin} --help [COMMAND]
+        let message = `\`\`\`sh-session
+$ ${config.bin} plugins:install ${config.name}`;
+        if (config.topics[0]) {
+            message += `\n$ ${config.bin} ${config.topics[0].name}:COMMAND
+running command...`;
+        }
+        message += `\n$ ${config.bin} plugins
+${config.name} ${process.env.OCLIF_NEXT_VERSION || config.version}`;
+        if (config.topics[0]) {
+            message += `\n$ ${config.bin} help ${config.topics[0].name}:COMMAND
 USAGE
-  $ ${config.bin} COMMAND
-...
-\`\`\`\n`,
-        ].join('\n').trim();
+  $ ${config.bin} ${config.topics[0].name}:COMMAND
+...`;
+        }
+        message += `\n\`\`\`\n`;
+        return [message].join('\n').trim();
     }
     multiCommands(config, commands, dir) {
         let topics = config.topics;
@@ -81,7 +84,7 @@ USAGE
             this.createTopicFile(path.join('.', dir, topic.name.replace(/:/g, '/') + '.md'), config, topic, commands.filter(c => c.id === topic.name || c.id.startsWith(topic.name + ':')));
         }
         return [
-            '# Command Topics\n',
+            '## Command Topics\n',
             ...topics.map(t => {
                 return util_1.compact([
                     `* [\`${config.bin} ${t.name}\`](${dir}/${t.name.replace(/:/g, '/')}.md)`,
@@ -106,7 +109,7 @@ USAGE
         return [
             ...commands.map(c => {
                 const usage = this.commandUsage(config, c);
-                return `* [\`${config.bin} ${usage}\`](#${slugify.slug(`${config.bin}-${usage}`)})`;
+                return `* [\`${config.bin} ${c.id}\`](#${slugify.slug(`${config.bin}-${c.id}`)})`;
             }),
             '',
             ...commands.map(c => this.renderCommand(config, c)).map(s => s.trim() + '\n'),
@@ -118,7 +121,7 @@ USAGE
         const HelpClass = plugin_help_1.getHelpClass(config);
         const help = new HelpClass(config, { stripAnsi: true, maxWidth: columns });
         const wrapper = new help_compatibility_1.HelpCompatibilityWrapper(help);
-        const header = () => `## \`${config.bin} ${this.commandUsage(config, c)}\``;
+        const header = () => `### \`${config.bin} ${c.id}\``;
         try {
             return util_1.compact([
                 header(),
@@ -197,7 +200,7 @@ USAGE
             return;
         p = p.replace(plugin.root + path.sep, '');
         if (plugin.pjson.devDependencies && plugin.pjson.devDependencies.typescript) {
-            p = p.replace(libRegex, 'src' + path.sep);
+            p = p.replace(libRegex, '');
             p = p.replace(/\.js$/, '.ts');
         }
         p = p.replace(/\\/g, '/'); // Replace windows '\' by '/'
@@ -225,9 +228,9 @@ USAGE
 exports.default = Readme;
 Readme.description = `adds commands to README.md in current directory
 The readme must have any of the following tags inside of it for it to be replaced or else it will do nothing:
-# Usage
+## Usage
 <!-- usage -->
-# Commands
+## Commands
 <!-- commands -->
 
 Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.
