diff --git a/node_modules/@salesforce/dev-scripts/bin/sf-clean.js b/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
index 07e8816..6b1d712 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
@@ -39,7 +39,7 @@ if (gitignorePath) {
 
   // The first line of the segment is what we are looking for. Either # -- CLEAN or # -- CLEAN ALL
   if (segments['CLEAN']) {
-    toClean = segments['CLEAN'];
+    toClean = toClean.concat(segments['CLEAN']);
   } else {
     const example = join(__dirname, '..', 'files', '.gitignore');
     log(
@@ -49,7 +49,7 @@ if (gitignorePath) {
     );
   }
   if (segments['CLEAN ALL']) {
-    toCleanAll = segments['CLEAN ALL'];
+    toCleanAll = toCleanAll.concat(segments['CLEAN ALL']);
   }
 }
 
@@ -58,5 +58,6 @@ if (cleanAll) {
   toClean = [...toClean, ...toCleanAll];
 }
 
+toClean = [...new Set(toClean)]
 log(`rm -rf ${toClean}`);
 shell.rm('-rf', toClean);
diff --git a/node_modules/@salesforce/dev-scripts/files/LICENSE.txt b/node_modules/@salesforce/dev-scripts/files/LICENSE.txt
index 9b64188..fbc4a64 100644
--- a/node_modules/@salesforce/dev-scripts/files/LICENSE.txt
+++ b/node_modules/@salesforce/dev-scripts/files/LICENSE.txt
@@ -1,4 +1,4 @@
-Copyright (c) 2018, Salesforce.com, Inc.
+Copyright (c) 2020, jayree
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
@@ -7,6 +7,6 @@ Redistribution and use in source and binary forms, with or without modification,
 
 * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
 
-* Neither the name of Salesforce.com nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+* Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
 
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
diff --git a/node_modules/@salesforce/dev-scripts/utils/sf-config.js b/node_modules/@salesforce/dev-scripts/utils/sf-config.js
index 8a09db4..615ff5d 100644
--- a/node_modules/@salesforce/dev-scripts/utils/sf-config.js
+++ b/node_modules/@salesforce/dev-scripts/utils/sf-config.js
@@ -21,6 +21,7 @@ const PACKAGE_DEFAULTS = {
     pretest: 'sf-compile-test',
     test: 'sf-test',
     lint: 'sf-lint',
+    'lint-fix': 'sf-lint --fix',
     prepack: 'sf-prepack',
   },
   husky: {
diff --git a/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js b/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
index 50ed4a0..d8e6432 100644
--- a/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
+++ b/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
@@ -42,36 +42,9 @@ module.exports = (packageRoot = require('./package-path'), inLernaProject) => {
     pjson.actions.push(`standardizing scripts: ${scriptsChanged.join(', ')}`);
   }
 
-  // Husky should only be at the root git project (the lerna project)
-  if (inLernaProject) {
-    if (pjson.contents.husky) {
-      pjson.actions.push('removing husky scripts; they should only be on the lerna package.json');
-      delete pjson.contents.husky;
-    }
-  } else {
-    // GENERATE HUSKY
-    const huskyList = Object.keys(config.husky);
-
-    if (huskyList.length > 0) {
-      pjson.actions.push(`standardizing husky: ${huskyList.join(', ')}`);
-
-      const husky = pjson.get('husky');
-
-      if (!husky.hooks) {
-        husky.hooks = {};
-      }
-
-      const standardizedHuskyScripts = [];
-      for (const [hookName, huskyScript] of Object.entries(config.husky)) {
-        if (husky.hooks[hookName] !== huskyScript) {
-          husky.hooks[hookName] = huskyScript;
-          standardizedHuskyScripts.push(hookName);
-        }
-      }
-      if (standardizedHuskyScripts.length > 0) {
-        pjson.actions.push(`standardize husky scripts ${standardizedHuskyScripts.join(', ')}`);
-      }
-    }
+  if (pjson.contents.husky) {
+    pjson.actions.push('migrate to husky 5, removing husky scripts.');
+    delete pjson.contents.husky;
   }
 
   try {
diff --git a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
index 26e2a0d..caae5b8 100644
--- a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
+++ b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
@@ -92,16 +92,16 @@ module.exports = (projectPath, inLernaProject) => {
   add('sinon');
   add('chai');
 
-  remove('@commitlint/cli');
-  remove('@commitlint/config-conventional');
-  remove('source-map-support');
-  remove('xunit-file');
-  remove('@types/chai');
-  remove('@types/mocha');
-  remove('@types/node');
-  remove('@types/sinon');
-  remove('typedoc');
-  remove('typedoc-plugin-external-module-name');
+  add('@commitlint/cli');
+  add('@commitlint/config-conventional');
+  add('source-map-support');
+  // remove('xunit-file');
+  add('@types/chai');
+  add('@types/mocha');
+  add('@types/node');
+  add('@types/sinon');
+  // remove('typedoc');
+  // remove('typedoc-plugin-external-module-name');
 
   // We use eslint now
   remove('tslint');
