diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
index d9f31b6..43e1026 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
@@ -10,7 +10,8 @@ const shell = require('../utils/shelljs');

 const exists = require('../utils/exists');
 const packageRoot = require('../utils/package-path');
-const tsc = require.resolve('typescript/bin/tsc');
+const path = require('path');
+const tsc = path.resolve(require.resolve('typescript'),'../../../','.bin','tsc');

 shell.exec(`${tsc} -p . --pretty`, {
   cwd: packageRoot
diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js
index 939048a..93626f9 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js
@@ -13,7 +13,8 @@ const shell = require('../utils/shelljs');
 const loadRootPath = require('../utils/load-root-path');
 const packageRoot = require('../utils/package-path');
 const { isMultiPackageProject } = require('../utils/project-type');
-const typedoc = require.resolve('typedoc/bin/typedoc');
+const path = require('path');
+const typedoc = path.resolve(require.resolve('typedoc'), '../../../', '.bin', 'typedoc');

 const options = require('@salesforce/dev-config/typedoc');

diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js
index 68dc22e..80f42b7 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js
@@ -11,7 +11,8 @@ const shell = require('../utils/shelljs');
 const exists = require('../utils/exists');
 const packageRoot = require('../utils/package-path');

-const lint = require.resolve('tslint/bin/tslint');
+const path = require('path');
+const lint = path.resolve(require.resolve('tslint'), '../../../', '.bin', 'tslint');

 const extras = process.argv.slice(2).join(' ');

diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
index 981a56d..65efd29 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
@@ -10,8 +10,8 @@ const shell = require('../utils/shelljs');

 const packageRoot = require('../utils/package-path');
 const SfdxDevConfig = require('../utils/sfdx-dev-config');
-
-const nyc = require.resolve('nyc/bin/nyc');
+const path = require('path');
+const nyc = path.resolve(require.resolve('nyc'), '../../', '.bin', 'nyc');

 const config = new SfdxDevConfig(packageRoot);
 const testConfig = config.get('test') || {};
diff --git a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
index 36c9e8c..ba3ff1d 100644
--- a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
+++ b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
@@ -11,14 +11,25 @@ const SfdxDevConfig = require('./sfdx-dev-config');

 module.exports = (projectPath, inLernaProject) => {
   const pjson = new PackageJson(projectPath);
+  const pjson_devconfig = new PackageJson(process.cwd() + '/node_modules/@salesforce/dev-config');
+

   log(`Checking dependencies for ${pjson.name}`);

   const config = new SfdxDevConfig(projectPath);
   const dependencies = pjson.get('devDependencies');
+  const dependencies_devconfig = pjson_devconfig.get('dependencies');

   const added = [];
   const removed = [];
+  const synced = [];
+
+  const sync = name => {
+    if (dependencies[name] !== dependencies_devconfig[name]) {
+      synced.push(name + ' ' + dependencies[name] + ' => ' + dependencies_devconfig[name]);
+      dependencies[name] = dependencies_devconfig[name];
+    }
+  };

   const remove = name => {
     if (dependencies[name]) {
@@ -40,12 +51,13 @@ module.exports = (projectPath, inLernaProject) => {

   // Included by dev-scripts
   remove('nyc');
-  remove('typescript');
+  // sync('typescript');
   remove('ts-node');
   remove('@commitlint/cli');
   remove('@commitlint/config-conventional');
   remove('source-map-support');
-  remove('tslint');
+  // sync('tslint');
+  // sync('tslint-microsoft-contrib');
   remove('xunit-file');
   remove('sinon');
   remove('mocha');
@@ -65,6 +77,11 @@ module.exports = (projectPath, inLernaProject) => {
     added.forEach(dep => log(dep, 2));
   }

+  if (synced.length > 0) {
+    log(`synced`);
+    synced.forEach(dep => log(dep, 2));
+  }
+
   pjson.write();
-  return added.length > 0;
+  return added.length > 0 || synced.length > 0;
 };
