diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
index d9f31b6..e7e71a8 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-compile.js
@@ -10,7 +10,8 @@ const shell = require('../utils/shelljs');
 
 const exists = require('../utils/exists');
 const packageRoot = require('../utils/package-path');
-const tsc = require.resolve('typescript/bin/tsc');
+const path = require('path');
+const tsc = path.resolve(packageRoot, 'node_modules/.bin/tsc')
 
 shell.exec(`${tsc} -p . --pretty`, {
   cwd: packageRoot
diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js
index 0d22aa7..5acbae1 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-docs.js
@@ -13,7 +13,8 @@ const shell = require('../utils/shelljs');
 const loadRootPath = require('../utils/load-root-path');
 const packageRoot = require('../utils/package-path');
 const { isMultiPackageProject } = require('../utils/project-type');
-const typedoc = require.resolve('typedoc/bin/typedoc');
+const path = require('path');
+const typedoc = path.resolve(packageRoot, 'node_modules/.bin/typedoc')
 
 const options = require('@salesforce/dev-config/typedoc');
 
diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-husky-pre-commit.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-husky-pre-commit.js
index 4484cdb..eb13e04 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-husky-pre-commit.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-husky-pre-commit.js
@@ -9,13 +9,12 @@
 const shell = require('../utils/shelljs');
 
 const packageRoot = require('../utils/package-path');
-const prettierConfig = require.resolve('@salesforce/dev-config/prettier');
 
 // double -- to pass through yarn then lerna
 shell.exec(`yarn lint`, {
   cwd: packageRoot
 });
 
-shell.exec(`pretty-quick --staged --config ${prettierConfig}`, {
+shell.exec(`pretty-quick --staged`, {
   cwd: packageRoot
 });
diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js
index 68dc22e..ff96758 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-lint.js
@@ -11,7 +11,8 @@ const shell = require('../utils/shelljs');
 const exists = require('../utils/exists');
 const packageRoot = require('../utils/package-path');
 
-const lint = require.resolve('tslint/bin/tslint');
+const path = require('path');
+const lint = path.resolve(packageRoot, 'node_modules/.bin/tslint')
 
 const extras = process.argv.slice(2).join(' ');
 
diff --git a/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js b/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
index 981a56d..0de9520 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sfdx-test.js
@@ -10,8 +10,8 @@ const shell = require('../utils/shelljs');
 
 const packageRoot = require('../utils/package-path');
 const SfdxDevConfig = require('../utils/sfdx-dev-config');
-
-const nyc = require.resolve('nyc/bin/nyc');
+const path = require('path');
+const nyc = path.resolve(packageRoot, 'node_modules/.bin/nyc')
 
 const config = new SfdxDevConfig(packageRoot);
 const testConfig = config.get('test') || {};
diff --git a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
index 36c9e8c..06de9ca 100644
--- a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
+++ b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
@@ -39,26 +39,27 @@ module.exports = (projectPath, inLernaProject) => {
   }
 
   // Included by dev-scripts
-  remove('nyc');
-  remove('typescript');
-  remove('ts-node');
-  remove('@commitlint/cli');
-  remove('@commitlint/config-conventional');
-  remove('source-map-support');
-  remove('tslint');
-  remove('xunit-file');
-  remove('sinon');
-  remove('mocha');
-  remove('chai');
-  remove('@types/chai');
-  remove('@types/mocha');
-  remove('@types/node');
-  remove('@types/sinon');
-  remove('typedoc');
-  remove('typedoc-plugin-external-module-name');
-  remove('prettier');
-  remove('pretty-quick');
-  remove('@salesforce/dev-config');
+  // remove('nyc');
+  // remove('typescript');
+  // remove('ts-node');
+  // remove('@commitlint/cli');
+  // remove('@commitlint/config-conventional');
+  // remove('source-map-support');
+  // remove('tslint');
+  // remove('tslint-microsoft-contrib');
+  // remove('xunit-file');
+  // remove('sinon');
+  // remove('mocha');
+  // remove('chai');
+  // remove('@types/chai');
+  // remove('@types/mocha');
+  // remove('@types/node');
+  // remove('@types/sinon');
+  // remove('typedoc');
+  // remove('typedoc-plugin-external-module-name');
+  // remove('prettier');
+  // remove('pretty-quick');
+  // remove('@salesforce/dev-config');
 
   if (added.length > 0) {
     log(`added`);
