diff --git a/node_modules/@salesforce/dev-scripts/bin/sf-clean.js b/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
index 657c525..95676aa 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
@@ -50,7 +50,7 @@ if (gitignorePath) {
 
   // The first line of the segment is what we are looking for. Either # -- CLEAN or # -- CLEAN ALL
   if (segments['CLEAN']) {
-    toClean = segments['CLEAN'];
+    toClean = toClean.concat(segments['CLEAN']);
   } else {
     const example = join(__dirname, '..', 'files', '.gitignore');
     log(
@@ -60,7 +60,7 @@ if (gitignorePath) {
     );
   }
   if (segments['CLEAN ALL']) {
-    toCleanAll = segments['CLEAN ALL'];
+    toCleanAll = toCleanAll.concat(segments['CLEAN ALL']);
   }
 }
 
@@ -73,5 +73,6 @@ if (ignoreSigningArtifacts) {
   toClean = toClean.filter((item) => !item.endsWith('.tgz') && !item.endsWith('.sig'));
 }
 
+toClean = [...new Set(toClean)]
 log(`rm -rf ${toClean}`);
 shell.rm('-rf', toClean);
diff --git a/node_modules/@salesforce/dev-scripts/bin/sf-install.js b/node_modules/@salesforce/dev-scripts/bin/sf-install.js
index 2e1a739..67d15b0 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sf-install.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sf-install.js
@@ -19,7 +19,8 @@ if (changed) {
 } else {
   require('../utils/standardize-pjson')();
   require('../utils/standardize-files')();
-  if (exists('.git')) {
+  const isCi = process.env.CI !== undefined;
+  if (exists('.git') && !isCi) {
     require('../utils/husky-init')();
   }
 }
diff --git a/node_modules/@salesforce/dev-scripts/files/eslintrc-strict.cjs b/node_modules/@salesforce/dev-scripts/files/eslintrc-strict.cjs
index 33b19f5..eac972b 100644
--- a/node_modules/@salesforce/dev-scripts/files/eslintrc-strict.cjs
+++ b/node_modules/@salesforce/dev-scripts/files/eslintrc-strict.cjs
@@ -9,5 +9,24 @@
 // See more at https://github.com/forcedotcom/sfdx-dev-packages/tree/master/packages/dev-scripts
 
 module.exports = {
-  extends: ['eslint-config-salesforce-typescript', 'eslint-config-salesforce-license', 'eslint-config-prettier'],
+  extends: ['eslint-config-salesforce-typescript', 'plugin:sf-plugin/migration', 'plugin:prettier/recommended'],
+  plugins: ['eslint-plugin-header'],
+  ignorePatterns: ['**/*.d.ts', '**/*.cjs'],
+  rules: {
+    'header/header': [
+      2,
+      'block',
+      [
+        '',
+        {
+          pattern: ' \\* Copyright \\(c\\) \\d{4}, jayree',
+          template: ' * Copyright (c) 2023, jayree',
+        },
+        ' * All rights reserved.',
+        ' * Licensed under the BSD 3-Clause license.',
+        ' * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause',
+        ' ',
+      ],
+    ],
+  },
 };
diff --git a/node_modules/@salesforce/dev-scripts/files/mocharc.json b/node_modules/@salesforce/dev-scripts/files/mocharc.json
index 2e6bae4..6725c65 100644
--- a/node_modules/@salesforce/dev-scripts/files/mocharc.json
+++ b/node_modules/@salesforce/dev-scripts/files/mocharc.json
@@ -3,5 +3,6 @@
   "watch-extensions": "ts",
   "recursive": true,
   "reporter": "spec",
-  "timeout": 5000
+  "timeout": 5000,
+  "node-option": ["experimental-specifier-resolution=node", "loader=ts-node/esm", "no-warnings"]
 }
diff --git a/node_modules/@salesforce/dev-scripts/files/tsconfig-strict-esm.json b/node_modules/@salesforce/dev-scripts/files/tsconfig-strict-esm.json
new file mode 100644
index 0000000..0e56a73
--- /dev/null
+++ b/node_modules/@salesforce/dev-scripts/files/tsconfig-strict-esm.json
@@ -0,0 +1,12 @@
+// Generated - Do not modify. Controlled by @salesforce/dev-scripts
+// See more at https://github.com/forcedotcom/sfdx-dev-packages/tree/master/packages/dev-scripts
+
+{
+  "extends": "@salesforce/dev-config/tsconfig-strict-esm",
+  "compilerOptions": {
+    "outDir": "lib",
+    "incremental": true,
+    "skipLibCheck": true
+  },
+  "include": ["./src/**/*.ts"]
+}
diff --git a/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict-esm.json b/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict-esm.json
new file mode 100644
index 0000000..5dece17
--- /dev/null
+++ b/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict-esm.json
@@ -0,0 +1,7 @@
+// Generated - Do not modify. Controlled by @salesforce/dev-scripts
+// See more at https://github.com/forcedotcom/sfdx-dev-packages/tree/master/packages/dev-scripts
+
+{
+  "extends": "@salesforce/dev-config/tsconfig-test-strict-esm",
+  "include": ["./**/*.ts"]
+}
diff --git a/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict.json b/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict.json
index 44147cc..87197ec 100644
--- a/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict.json
+++ b/node_modules/@salesforce/dev-scripts/files/tsconfig-test-strict.json
@@ -3,5 +3,11 @@
 
 {
   "extends": "@salesforce/dev-config/tsconfig-test-strict",
+  "compilerOptions": {
+    "skipLibCheck": true,
+    "allowSyntheticDefaultImports": true,
+    "module": "node16",
+    "moduleResolution": "node16"
+  },
   "include": ["./**/*.ts"]
 }
diff --git a/node_modules/@salesforce/dev-scripts/utils/package-json.js b/node_modules/@salesforce/dev-scripts/utils/package-json.js
index c1d55c8..a33594b 100644
--- a/node_modules/@salesforce/dev-scripts/utils/package-json.js
+++ b/node_modules/@salesforce/dev-scripts/utils/package-json.js
@@ -25,6 +25,9 @@ class PackageJson {
     if (this.contents.scripts) {
       this.contents.scripts = orderMap(this.contents.scripts);
     }
+    if (this.contents.wireit) {
+      this.contents.wireit = orderMap(this.contents.wireit);
+    }
     if (this.contents.devDependencies) {
       this.contents.devDependencies = orderMap(this.contents.devDependencies);
     }
@@ -41,6 +44,7 @@ class PackageJson {
 
       writeFileSync(this.pjsonPath, pjson);
       log(`wrote changes to ${this.pjsonPath}`, 1);
+      process.exit(1);
     }
   }
 
@@ -53,6 +57,18 @@ class PackageJson {
     }
     return this.contents[name];
   }
+
+  delete(name, value) {
+    if (!name) {
+      throw new Error('property name is required');
+    }
+    if (!value) {
+      throw new Error('property value is required');
+    }
+    if (this.contents[name][value]) {
+      delete this.contents[name][value];
+    }
+  }
 }
 
 module.exports = PackageJson;
diff --git a/node_modules/@salesforce/dev-scripts/utils/sf-config.js b/node_modules/@salesforce/dev-scripts/utils/sf-config.js
index 8d1bc90..ff2b105 100644
--- a/node_modules/@salesforce/dev-scripts/utils/sf-config.js
+++ b/node_modules/@salesforce/dev-scripts/utils/sf-config.js
@@ -33,7 +33,7 @@ const PACKAGE_DEFAULTS = {
       dependencies: ['compile', 'lint'],
     },
     compile: {
-      command: 'tsc -p . --pretty --incremental',
+      command: 'tsc --build --pretty',
       files: ['src/**/*.ts', '**/tsconfig.json', 'messages/**'],
       output: ['lib/**', '*.tsbuildinfo'],
       clean: 'if-file-deleted',
@@ -44,7 +44,7 @@ const PACKAGE_DEFAULTS = {
       output: [],
     },
     lint: {
-      command: 'eslint src test --color --cache --cache-location .eslintcache',
+      command: 'eslint --color --cache --cache-location .eslintcache .',
       files: ['src/**/*.ts', 'test/**/*.ts', 'messages/**', '**/.eslint*', '**/tsconfig.json'],
       output: [],
     },
@@ -65,7 +65,7 @@ const PACKAGE_DEFAULTS = {
       dependencies: ['test:only', 'test:compile', 'link-check'],
     },
     'test:only': {
-      command: 'nyc mocha "test/**/*.test.ts"',
+      command: 'c8 mocha "test/**/*.test.ts"',
       // things that use `chalk` might not output colors with how wireit uses spawn and gha treats that as non-tty
       // see https://github.com/chalk/supports-color/issues/106
       env: {
@@ -144,7 +144,7 @@ const resolveConfig = (path) => {
   const configFromFile = (result && result.config) || {};
 
   if (configFromFile.test?.testsPath) {
-    defaults.wireit['test:only'].command = `nyc mocha "${configFromFile.test.testsPath}"`;
+    defaults.wireit['test:only'].command = `c8 mocha "${configFromFile.test.testsPath}"`;
   }
 
   // Allow users to override certain scripts
@@ -171,6 +171,12 @@ const resolveConfig = (path) => {
     delete config.wireit[scriptName];
   });
 
+  Object.keys(config.wireit).forEach((scriptName) => {
+    if (config.wireit[scriptName].dependencies) {
+      config.wireit[scriptName].dependencies = config.wireit[scriptName].dependencies.filter(depScriptName => !config['exclude-scripts']?.includes(depScriptName));
+    }
+  })
+
   resolvedConfigs[path] = config;
   return config;
 };
diff --git a/node_modules/@salesforce/dev-scripts/utils/standardize-files.js b/node_modules/@salesforce/dev-scripts/utils/standardize-files.js
index b47f4f9..0d69e29 100644
--- a/node_modules/@salesforce/dev-scripts/utils/standardize-files.js
+++ b/node_modules/@salesforce/dev-scripts/utils/standardize-files.js
@@ -19,6 +19,9 @@ const FILE_NAME_LICENSE = 'LICENSE.txt';
 const FILE_NAME_GITIGNORE = 'gitignore';
 const FILE_NAME_MOCHARC = 'mocharc.json';
 
+let config;
+let pkgRoot;
+
 // We don't copy over the .gitignore file since plugins/libraries might have their own
 // unique additions to the file. So in order to programmatically add entries, we need to
 // read the existing .gitignore file and append these entries to it in case they don't exist.
@@ -51,7 +54,7 @@ function copyFile(sourcePath, targetPath, override = false) {
   const fileExists = exists(targetPath);
   const shouldWriteFile = override || !fileExists;
 
-  if (shouldWriteFile && isDifferent(sourcePath, targetPath)) {
+  if (shouldWriteFile && isDifferent(sourcePath, targetPath) && !(config['exclude-files'] || []).some((e) => targetPath === join(pkgRoot,e))) {
     copyFileSync(sourcePath, targetPath);
     return targetPath;
   }
@@ -173,7 +176,8 @@ function replaceInFile(filePath, replaceFn) {
 
 // eslint-disable-next-line complexity
 module.exports = (packageRoot = require('./package-path')) => {
-  const config = resolveConfig(packageRoot);
+  config = resolveConfig(packageRoot);
+  pkgRoot = packageRoot;
   const testPath = join(packageRoot, 'test');
   const scripts = config.scripts;
 
@@ -252,12 +256,12 @@ module.exports = (packageRoot = require('./package-path')) => {
     const compileConfig = config.compile || {};
     const strict = config.strict || compileConfig.strict;
 
-    const tsconfigSourcePath = join(FILES_PATH, strict ? 'tsconfig-strict.json' : 'tsconfig.json');
+    const tsconfigSourcePath = join(FILES_PATH, strict ? 'tsconfig-strict-esm.json' : 'tsconfig.json');
     const tsconfigTargetPath = join(packageRoot, 'tsconfig.json');
     added.push(copyFile(tsconfigSourcePath, tsconfigTargetPath, strict));
 
     if (exists(testPath)) {
-      const tsconfigTestSourcePath = join(FILES_PATH, strict ? 'tsconfig-test-strict.json' : 'tsconfig-test.json');
+      const tsconfigTestSourcePath = join(FILES_PATH, strict ? 'tsconfig-test-strict-esm.json' : 'tsconfig-test.json');
       const tsconfigTestTargetPath = join(testPath, 'tsconfig.json');
       added.push(copyFile(tsconfigTestSourcePath, tsconfigTestTargetPath, strict));
     }
diff --git a/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js b/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
index 3001ff7..6e422c3 100644
--- a/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
+++ b/node_modules/@salesforce/dev-scripts/utils/standardize-pjson.js
@@ -45,27 +45,34 @@ module.exports = (packageRoot = require('./package-path')) => {
   const scriptList = Object.entries(config.scripts);
   const wireitList = Object.entries(config.wireit);
 
+  let excludeScripts = config['exclude-scripts'] || [];
+
+  excludeScripts.forEach((scriptName) => {
+    pjson.delete('scripts',scriptName)
+    pjson.delete('wireit',scriptName)
+  });
+
   if (scriptList.length > 0) {
     const scriptsChanged = [];
 
     const scripts = pjson.get('scripts');
     // eslint-disable-next-line prefer-const
     for (let [scriptName, scriptCommand] of scriptList) {
-      if (scripts[scriptName] !== scriptCommand) {
+      if (JSON.stringify(scripts[scriptName]) !== JSON.stringify(scriptCommand)) {
         scripts[scriptName] = scriptCommand;
         scriptsChanged.push(scriptName);
       }
     }
-    pjson.actions.push(`standardizing scripts: ${scriptsChanged.join(', ')}`);
     if (wireitList.length > 0) {
       const wireit = pjson.get('wireit');
       for (const [scriptName, scriptCommand] of wireitList) {
-        if (wireit[scriptName] !== scriptCommand) {
+        if (JSON.stringify(wireit[scriptName]) !== JSON.stringify(scriptCommand)) {
           wireit[scriptName] = scriptCommand;
           scriptsChanged.push(scriptName);
         }
       }
     }
+    pjson.actions.push(`standardizing scripts: ${scriptsChanged.join(', ')}`);
   }
 
   try {
diff --git a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
index 46a5477..e2510ce 100644
--- a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
+++ b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
@@ -116,7 +116,7 @@ module.exports = (projectPath) => {
   /**
    * We don't want these in any repo.  This is a good way to clean up things en masse
    */
-  const bannedDeps = ['cz-conventional-changelog', 'lint-staged', 'tslint', 'eslint-plugin-prettier'].concat(
+  const bannedDeps = ['cz-conventional-changelog', 'lint-staged', 'tslint'].concat(
     scripts.format ? [] : ['prettier', '@salesforce/prettier-config']
   );
 
