diff --git a/node_modules/@salesforce/dev-scripts/bin/sf-clean.js b/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
index 07e8816..88bab29 100755
--- a/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
+++ b/node_modules/@salesforce/dev-scripts/bin/sf-clean.js
@@ -39,7 +39,7 @@ if (gitignorePath) {
 
   // The first line of the segment is what we are looking for. Either # -- CLEAN or # -- CLEAN ALL
   if (segments['CLEAN']) {
-    toClean = segments['CLEAN'];
+    toClean = toClean.concat(segments['CLEAN']);
   } else {
     const example = join(__dirname, '..', 'files', '.gitignore');
     log(
@@ -49,7 +49,7 @@ if (gitignorePath) {
     );
   }
   if (segments['CLEAN ALL']) {
-    toCleanAll = segments['CLEAN ALL'];
+    toCleanAll = toCleanAll.concat(segments['CLEAN ALL']);
   }
 }
 
@@ -57,6 +57,6 @@ if (gitignorePath) {
 if (cleanAll) {
   toClean = [...toClean, ...toCleanAll];
 }
-
+toClean = [...new Set(toClean)]
 log(`rm -rf ${toClean}`);
 shell.rm('-rf', toClean);
diff --git a/node_modules/@salesforce/dev-scripts/files/LICENSE.txt b/node_modules/@salesforce/dev-scripts/files/LICENSE.txt
index 026246e..b146616 100644
--- a/node_modules/@salesforce/dev-scripts/files/LICENSE.txt
+++ b/node_modules/@salesforce/dev-scripts/files/LICENSE.txt
@@ -1,4 +1,4 @@
-Copyright (c) REPLACE_YEAR, Salesforce.com, Inc.
+Copyright (c) REPLACE_YEAR, jayree
 All rights reserved.
 
 Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
@@ -7,6 +7,6 @@ Redistribution and use in source and binary forms, with or without modification,
 
 * Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
 
-* Neither the name of Salesforce.com nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+* Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
 
 THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
\ No newline at end of file
diff --git a/node_modules/@salesforce/dev-scripts/files/eslintrc-test.js b/node_modules/@salesforce/dev-scripts/files/eslintrc-test.json
similarity index 61%
rename from node_modules/@salesforce/dev-scripts/files/eslintrc-test.js
rename to node_modules/@salesforce/dev-scripts/files/eslintrc-test.json
index 79fd9f4..b8b42ab 100644
--- a/node_modules/@salesforce/dev-scripts/files/eslintrc-test.js
+++ b/node_modules/@salesforce/dev-scripts/files/eslintrc-test.json
@@ -1,25 +1,25 @@
 /*
- * Copyright (c) 2020, salesforce.com, inc.
+ * Copyright (c) 2022, jayree
  * All rights reserved.
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
-module.exports = {
-  extends: '../.eslintrc.js',
+{
+  "extends": "../.eslintrc.json",
   // Allow describe and it
-  env: { mocha: true },
-  rules: {
+  "env": { "mocha": true },
+  "rules": {
     // Allow assert style expressions. i.e. expect(true).to.be.true
-    'no-unused-expressions': 'off',
+    "no-unused-expressions": "off",
 
     // It is common for tests to stub out method.
 
     // Return types are defined by the source code. Allows for quick overwrites.
-    '@typescript-eslint/explicit-function-return-type': 'off',
+    "@typescript-eslint/explicit-function-return-type": "off",
     // Mocked out the methods that shouldn't do anything in the tests.
-    '@typescript-eslint/no-empty-function': 'off',
+    "@typescript-eslint/no-empty-function": "off",
     // Easily return a promise in a mocked method.
-    '@typescript-eslint/require-await': 'off',
-  },
-};
+    "@typescript-eslint/require-await": "off"
+  }
+}
diff --git a/node_modules/@salesforce/dev-scripts/files/eslintrc.js b/node_modules/@salesforce/dev-scripts/files/eslintrc.js
deleted file mode 100644
index d4166fd..0000000
--- a/node_modules/@salesforce/dev-scripts/files/eslintrc.js
+++ /dev/null
@@ -1,9 +0,0 @@
-/*
- * Copyright (c) 2020, salesforce.com, inc.
- * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-module.exports = {
-  extends: ['eslint-config-salesforce-typescript', 'eslint-config-salesforce-license', 'eslint-config-prettier'],
-};
diff --git a/node_modules/@salesforce/dev-scripts/files/eslintrc.json b/node_modules/@salesforce/dev-scripts/files/eslintrc.json
new file mode 100644
index 0000000..dafb62c
--- /dev/null
+++ b/node_modules/@salesforce/dev-scripts/files/eslintrc.json
@@ -0,0 +1,28 @@
+/*
+ * Copyright (c) 2022, jayree
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+{
+  "plugins": ["eslint-plugin-header"],
+  "extends": ["eslint-config-salesforce-typescript", "eslint-config-prettier", "plugin:sf-plugin/recommended"],
+  "rules": {
+    "header/header": [
+      2,
+      "block",
+      [
+        "",
+        {
+          "pattern": " \\* Copyright \\(c\\) \\d{4}, jayree",
+          "template": " * Copyright (c) 2022, jayree"
+        },
+        " * All rights reserved.",
+        " * Licensed under the BSD 3-Clause license.",
+        " * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause",
+        " "
+      ]
+    ]
+  }
+}
diff --git a/node_modules/@salesforce/dev-scripts/files/gitignore b/node_modules/@salesforce/dev-scripts/files/gitignore
index a8d60c7..05b2bab 100644
--- a/node_modules/@salesforce/dev-scripts/files/gitignore
+++ b/node_modules/@salesforce/dev-scripts/files/gitignore
@@ -12,7 +12,7 @@ yarn-error.log
 lerna-debug.log
 
 # compile source
-lib
+# lib
 
 # test artifacts
 *xunit.xml
diff --git a/node_modules/@salesforce/dev-scripts/package.json b/node_modules/@salesforce/dev-scripts/package.json
index cf78b1f..39e8f4f 100644
--- a/node_modules/@salesforce/dev-scripts/package.json
+++ b/node_modules/@salesforce/dev-scripts/package.json
@@ -58,6 +58,7 @@
     "eslint-plugin-header": "^3.0.0",
     "eslint-plugin-import": "2.26.0",
     "eslint-plugin-jsdoc": "^35.1.2",
+    "eslint-plugin-prettier": "^4.2.1",
     "eslint-plugin-prefer-arrow": "^1.2.1",
     "husky": "^7.0.4",
     "mocha": "^9.1.3",
diff --git a/node_modules/@salesforce/dev-scripts/utils/sf-config.js b/node_modules/@salesforce/dev-scripts/utils/sf-config.js
index d3870e6..4585630 100644
--- a/node_modules/@salesforce/dev-scripts/utils/sf-config.js
+++ b/node_modules/@salesforce/dev-scripts/utils/sf-config.js
@@ -21,6 +21,7 @@ const PACKAGE_DEFAULTS = {
     pretest: 'sf-compile-test',
     test: 'sf-test',
     lint: 'sf-lint',
+    'lint-fix': 'sf-lint --fix',
     prepack: 'sf-prepack',
   },
 };
diff --git a/node_modules/@salesforce/dev-scripts/utils/standardize-files.js b/node_modules/@salesforce/dev-scripts/utils/standardize-files.js
index 0142a61..702b951 100644
--- a/node_modules/@salesforce/dev-scripts/utils/standardize-files.js
+++ b/node_modules/@salesforce/dev-scripts/utils/standardize-files.js
@@ -34,7 +34,7 @@ function isDifferent(sourcePath, targetPath) {
   return true;
 }
 
-function copyFile(sourcePath, targetPath, override = false) {
+function copyFile(sourcePath, targetPath, override = true) {
   const fileExists = exists(targetPath);
   const shouldWriteFile = override || !fileExists;
 
@@ -121,13 +121,13 @@ module.exports = (packageRoot = require('./package-path'), inLernaProject) => {
     const lintConfig = config.lint || {};
     const strict = config.strict || lintConfig.strict;
 
-    const eslintSourcePath = join(FILES_PATH, strict ? 'eslintrc-strict.js' : 'eslintrc.js');
-    const eslintTargetPath = join(packageRoot, '.eslintrc.js');
+    const eslintSourcePath = join(FILES_PATH, strict ? 'eslintrc-strict.js' : 'eslintrc.json');
+    const eslintTargetPath = join(packageRoot, '.eslintrc.json');
     added.push(copyFile(eslintSourcePath, eslintTargetPath, strict));
 
     if (exists(testPath)) {
-      const eslintTestSourcePath = join(FILES_PATH, strict ? 'eslintrc-test-strict.js' : 'eslintrc-test.js');
-      const eslintTestTargetPath = join(testPath, '.eslintrc.js');
+      const eslintTestSourcePath = join(FILES_PATH, strict ? 'eslintrc-test-strict.js' : 'eslintrc-test.json');
+      const eslintTestTargetPath = join(testPath, '.eslintrc.json');
       added.push(copyFile(eslintTestSourcePath, eslintTestTargetPath, strict));
     }
 
diff --git a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
index 8801aa9..6559f22 100644
--- a/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
+++ b/node_modules/@salesforce/dev-scripts/utils/write-dependencies.js
@@ -16,7 +16,7 @@ module.exports = (projectPath, inLernaProject) => {
   const config = resolveConfig(projectPath, inLernaProject);
   const dependencies = pjson.get('devDependencies');
 
-  const added = [];
+  let added = [];
   const removed = [];
 
   const getVersionNum = (ver) => (ver.startsWith('^') || ver.startsWith('~') ? ver.slice(1) : ver);
@@ -94,16 +94,16 @@ module.exports = (projectPath, inLernaProject) => {
   add('sinon');
   add('chai');
 
-  remove('@commitlint/cli');
-  remove('@commitlint/config-conventional');
-  remove('source-map-support');
-  remove('@types/chai');
-  remove('@types/mocha');
-  remove('@types/node');
-  remove('@types/sinon');
+  add('@commitlint/cli');
+  add('@commitlint/config-conventional');
+  add('source-map-support');
+  add('@types/chai');
+  add('@types/mocha');
+  add('@types/node');
+  add('@types/sinon');
   remove('typedoc');
   remove('typedoc-plugin-missing-exports');
-  remove('eslint-plugin-prettier');
+  add('eslint-plugin-prettier');
   remove('lint-staged');
   remove('cz-conventional-changelog');
   // We use eslint now
@@ -126,6 +126,9 @@ module.exports = (projectPath, inLernaProject) => {
     Object.entries(eslintHeaderPjson.devDependencies).forEach(([name, version]) => add(name, version));
   }
 
+  remove('eslint-config-salesforce-license');
+  added = added.filter(item => item !== 'eslint-config-salesforce-license')
+
   if (added.length > 0) {
     pjson.actions.push(`adding required devDependencies ${added.join(', ')}`);
   }
