diff --git a/.eslintrc.js b/.eslintrc.js
index 4ab8a1b1d5d260bfba85b3c45fc6488b50fe260d..122604391aad5db6335e6c56c4d44ecac788630e 100644
--- a/.eslintrc.js
+++ b/.eslintrc.js
@@ -7,8 +7,8 @@ module.exports = {
       [
         '',
         {
-          pattern: ' \\* Copyright \\(c\\) \\d{4}, salesforce\\.com, inc\\.',
-          template: ' * Copyright (c) 2020, salesforce.com, inc.',
+          pattern: ' \\* Copyright \\(c\\) \\d{4}, jayree',
+          template: ' * Copyright (c) 2020, jayree',
         },
         ' * All rights reserved.',
         ' * Licensed under the BSD 3-Clause license.',
