diff --git a/node_modules/oclif/lib/commands/readme.js b/node_modules/oclif/lib/commands/readme.js
index 48cb3c4..3c411b7 100644
--- a/node_modules/oclif/lib/commands/readme.js
+++ b/node_modules/oclif/lib/commands/readme.js
@@ -51,8 +51,8 @@ class Readme extends core_1.Command {
         return readme.replace(`<!-- ${tag} -->`, `<!-- ${tag} -->\n${body}\n<!-- ${tag}stop -->`);
     }
     toc(__, readme) {
-        return readme.split('\n').filter(l => l.startsWith('# '))
-            .map(l => l.trim().slice(2))
+        return readme.split('\n').filter(l => l.startsWith('## '))
+            .map(l => l.trim().slice(3))
             .map(l => `* [${l}](#${slugify.slug(l)})`)
             .join('\n');
     }
@@ -62,15 +62,9 @@ class Readme extends core_1.Command {
         const versionFlagsString = `(${versionFlags.join('|')})`;
         return [
             `\`\`\`sh-session
-$ npm install -g ${config.name}
-$ ${config.bin} COMMAND
-running command...
-$ ${config.bin} ${versionFlagsString}
-${config.name}/${process.env.OCLIF_NEXT_VERSION || config.version} ${process.platform}-${process.arch} node-v${process.versions.node}
-$ ${config.bin} --help [COMMAND]
-USAGE
-  $ ${config.bin} COMMAND
-...
+$ sfdx plugins:install ${config.name}${config.topics[0] ? `\n$ sfdx ${config.topics[0].name}:[COMMAND]\nrunning command...` : '' }
+$ ${config.bin} plugins
+${config.name} ${process.env.OCLIF_NEXT_VERSION || config.version}${config.topics[0] ? `\n$ ${config.bin} help ${config.topics[0].name}:[COMMAND]\nUSAGE\n  $ ${config.bin} ${config.topics[0].name}:COMMAND\n...` : '' }
 \`\`\`\n`,
         ].join('\n').trim();
     }
@@ -109,7 +103,7 @@ USAGE
         return [
             ...commands.map(c => {
                 const usage = this.commandUsage(config, c);
-                return usage ? `* [\`${config.bin} ${usage}\`](#${slugify.slug(`${config.bin}-${usage}`)})` : `* [\`${config.bin}\`](#${slugify.slug(`${config.bin}`)})`;
+                return usage ? `* [\`${config.bin} ${c.id}\`](#${slugify.slug(`${config.bin}-${c.id}`)})` : `* [\`${config.bin}\`](#${slugify.slug(`${config.bin}`)})`;
             }),
             '',
             ...commands.map(c => this.renderCommand(config, c)).map(s => s.trim() + '\n'),
@@ -122,7 +116,7 @@ USAGE
         const wrapper = new help_compatibility_1.HelpCompatibilityWrapper(help);
         const header = () => {
             const usage = this.commandUsage(config, c);
-            return usage ? `## \`${config.bin} ${usage}\`` : `## \`${config.bin}\``;
+            return usage ? `### \`${config.bin} ${c.id}\`` : `### \`${config.bin}\``;
         };
         try {
             return (0, util_1.compact)([
@@ -229,11 +223,11 @@ USAGE
 exports.default = Readme;
 Readme.description = `adds commands to README.md in current directory
 The readme must have any of the following tags inside of it for it to be replaced or else it will do nothing:
-# Usage
+## Usage
 <!-- usage -->
-# Commands
+## Commands
 <!-- commands -->
-# Table of contents
+## Table of contents
 <!-- toc -->
 
 Customize the code URL prefix by setting oclif.repositoryPrefix in package.json.
