diff --git a/node_modules/@oclif/dev-cli/lib/commands/readme.js b/node_modules/@oclif/dev-cli/lib/commands/readme.js
index d3abe19..71aa701 100644
--- a/node_modules/@oclif/dev-cli/lib/commands/readme.js
+++ b/node_modules/@oclif/dev-cli/lib/commands/readme.js
@@ -59,14 +59,14 @@ class Readme extends command_1.Command {
     usage(config) {
         return [
             `\`\`\`sh-session
-$ npm install -g ${config.name}
-$ ${config.bin} COMMAND
+$ ${config.bin} plugins:install ${config.name}
+$ ${config.bin} ${config.topics[0].name}:COMMAND
 running command...
-$ ${config.bin} (-v|--version|version)
-${config.name}/${process.env.OCLIF_NEXT_VERSION || config.version} ${process.platform}-${process.arch} node-v${process.versions.node}
-$ ${config.bin} --help [COMMAND]
+$ ${config.bin} plugins
+${config.name} ${process.env.OCLIF_NEXT_VERSION || config.version}
+$ ${config.bin} help ${config.topics[0].name}:COMMAND
 USAGE
-  $ ${config.bin} COMMAND
+  $ ${config.bin} ${config.topics[0].name}:COMMAND
 ...
 \`\`\`\n`,
         ].join('\n').trim();
@@ -189,7 +189,7 @@ USAGE
             return;
         p = p.replace(plugin.root + path.sep, '');
         if (plugin.pjson.devDependencies.typescript) {
-            p = p.replace(libRegex, 'src' + path.sep);
+            p = p.replace(libRegex, '');
             p = p.replace(/\.js$/, '.ts');
         }
         return p;
